kubernetes-elasticsearch-cluster/deploy.sh

```diff
@@ -21,6 +21,9 @@ if [ "$java_opts" != "null" ] && [ ! -z "$java_opts" ]; then
 	sed -i -e "s/-Xms1g -Xmx1g/$java_opts/g" es-data-stateful-deploy.yaml
 fi
 
+# Create the ssd storage class if it doesn't exist.
```

Comment:
Unrelated to this PR, but do you mind adding these commands to the top of this file? I realized these commands are missing when I recreated the cluster.

gcloud config set project PROJECT
gcloud container clusters get-credentials elasticsearch-cluster --zone ZONE